Project-1/README.md

```diff
@@ -63,60 +68,256 @@ T	T	F	T	T
+11.				Show: Qx   
+12.				~Qx	  			        Ass ID
+13.				~Px V ~Qx			      12, Add
+14.								            6, 13, ID
+15.				Px ∧ Qx	   			    7, 11, Adj
+16.						                4, 15, ID
+17.							              5, CD
+18.								            3, UD
+19.							              2, UD
+
+(b)
+1. Show: ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+2.	Show: ∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+3. 		Show: ¬(Px ∨ Qx) → (¬Px ∧ ¬Qx)
+4. 		¬(Px ∨ Qx)					            Ass CD
+5.			Show: (¬Px ∧ ¬Qx)
+6.			~(¬Px ∧ ¬Qx)			           	Ass ID
+7.				Show: ~Px
+8.				Px				                  Ass ID, DN
+9.				Px V Qx				              8, Add
+10.								                    4, 9 ID
+11. 				Show: ~Qx
+12.				      Qx				            Ass ID, DN
+13.			        	Px V Qx				      12, Add
+14.								                    4, 13, ID
+15.			         ~Px ∧ ¬Qx  	    		7, 11, Add
+16.								                    15, DD
+17. 							                  	5, CD
+18.							                     	3, UD
+19.								                    2, UD
+(c)   
+1. Show: ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+2. 	Show: ∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+3. 		Show: (¬Px ∨ ¬Qx) → ¬(Px ∧ Qx)
+4. 		(¬Px ∨ ¬Qx)					              Ass CD
+5. 			Show: ¬(Px ∧ Qx)
+6. 			(Px ∧ Qx)				                Ass ID, DN
+7. 			Px					                    6, S
+8.			~~Px					                  7, DN
+9.			~Qx					                    4, 8, MTP
+10.			Qx				                    	6, S
+11.								                      9, 10, ID
+12. 								                    5, CD
+13. 							                    	3, UD
+14.								                      2, UD
+(d)
+1. Show: ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+2. 	Show: ∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+3. 		Show: (¬Px ∧ ¬Qx) → ¬(Px ∨ Qx)
+4.		(¬Px ∧ ¬Qx)					              Ass CD
+5.		~Px						                    4, S
+6.		~Qx					                     	4, S
+7.			Show: ¬(Px ∨ Qx)
+8.			(Px ∨ Qx)				               Ass ID, DN
+9.			Qx					                   5, 8, MTP
+10.								                     6, 9, ID
+11.								                     7, CD
+12.								                     3, UD
+13.							                       2, UD
 
 Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs. 
 
+Well firsty and perhaps most notably the tree proof software provides no annotations (justifications?) for the how each line is derived from the line before it. I personally like having the annotations as it shows clearly how to move from one step to the next. Further, the tree proofs look like they start by universally instainting whereas in the natural deduction proofs you can go several route but the easiest with Universal Quantifiers is usually a universal derivation. I myself am prone to doing indirect derivations. This is because if it can be derived you can always(?) find a contradiction. Obviously, I still assumed inderictly once all of the unviersal were removed from the situation to complete the derivations. 
```

Comment:
typo inderictly/indirectly
instainting/instantiating
unviersal /universals

Not sure how it relates to what you were writing, but the main difference I found is that trees can only prove by contradiction/by providing a countermodel and follow one set algorithm, while with natural deduction you can do many different things, including but not limited to indirect proof.